main.go

```diff
@@ -232,6 +232,8 @@ func main() { //nolint:funlen,maintidx
 			},
 			// for prometheus and black-box deployment and ones we owns
 			&appsv1.Deployment{}: {Namespaces: deploymentCache},
+			// kueue need prometheusrules
+			&promv1.PrometheusRule{}: {},
```

Comment:
since there's no specific config, is that actually needed ?

Reply:
i will need to add the same deploymentCache here